src/Spatial/Euclidean/CoordinateSystem.cs

```diff
@@ -258,13 +258,44 @@ public static CoordinateSystem Rotation(Angle angle, Vector3D v)
         /// <returns>A rotated coordinate system</returns>
         public static CoordinateSystem Rotation(Angle yaw, Angle pitch, Angle roll)
```

Comment:
There is no need to keep the slow version around, now that we have a faster version which returns the same result. Please remove it.